README.md

```diff
@@ -427,6 +439,43 @@ Check out the implementations that the community has already produced [here](htt
 
 --------------------------------------------------------
 
+<a name="promises"></a>
+Promise Support
+---------------
+
+Each function taking a callback also can be used as a promise, if the callback is omitted. This applies for:
+
+- `db.get(key[, options])`
+- `db.put(key, value[, options])`
+- `db.del(key[, options])`
+- `db.batch(ops[, options])`
+- `db.batch().write()`
+
+The only exception is the `levelup` constructor itself, which if no callback is passed will lazily open the database backend in the background.
+
+Example:
+
+```js
+var db = levelup(leveldown('./my-db'))
+
+db.put('foo', 'bar')
+  .then(function () { return db.get('foo') })
+  .then(function (value) { console.log(value) })
```

Comment:
Let's add a `catch` handler too, for completeness and advocating best practices :)

Reply:
addressed